examples/ci_cd/gitlab-ci-cd/README.md

```diff
@@ -1,30 +1,61 @@
-
+## Step 2: Create a new git repo in your GitLab group and copy files from examples/advanced/gitlab-ci-cd folder to the root of your new GitLab repo
     cd examples/advanced/gitlab-ci-cd
-    gitlab-ci-cd
+    cp . $YOUR_GITLAB_REPO_ROOT
 
-## Step3: Update CI/CD settings config
- - Login to gitlab console, click on your repo and navigate to `settings/ci_cd`
- - Add the following variables before triggering the pipeline
+## Step 3: Update project settings-> CI/CD ->Variables
+ - Login to the GitLab console, Open your repo and navigate to `settings->ci_cd->Variables`
+ - Update the following variables as Key Value pairs  before triggering the pipeline
 
         AWS_ACCESS_KEY_ID           e.g., access key from devops admin iam role
         AWS_SECRET_ACCESS_KEY       e.g., secret key from devops admin iam role
         AWS_REGION                  e.g., eu-west-1
-        GITLAB_BASE_URL             e.g., https://gitlab.aws.dev/api/v4/
-        GITLAB_TOKEN                e.g., gitlab access token  
+        GITLAB_BASE_URL             e.g., https://gitlab.example.com/api/v4/
+        GITLAB_TOKEN                e.g., GitLab access token
+
+## Step 4: Update variables in input.tfvars file   
+   1. Update tenant,environment,zone as per your requirement
+   2. Update kubernetes_version to any version > "1.20"
+   3. Update CIDR of your VPC, vpc_cidcr = "10.2.0.0/16"
+
+
+## Step5: Commit changes and push to verify the pipeline
+Manually trigger the `tf-apply` to provision the resources
+
+## Step6: Verify the state file is updated in  your project (Infrastructure->Terraform-states) 
+
+## Step7: (Optional)  Manually Install, Configure and Run GitLab Agent for Kubernetes (“Agent”, for short) is your active in-cluster.
+This is for or connecting Kubernetes clusters to GitLab. Refer https://docs.gitlab.com/ee/user/clusters/agent/install/
+## Step8: Cleanup the deployed resources
+Manually trigger the `tf-destroy` stage in the GitLab Ci/CD pipeline to destroy your deployment.
+
+## Troubleshooting:
+
+- ### 403 Error when creating resource gitlab_group_cluster.aws_cluster
+
+    - Make sure the GitLab group that you're referencing in `data.gitlab_group.gitops-eks` exists and that you have access to it.
+
+- ### 400 Error when creating resource
+
+    - If the error contains `{message: {environment_scope: [cannot add duplicated environment scope]}}`, it is likely that an existing Kubernetes integration with the same environment scope was not removed. Remove any Kubernetes clusters with the same environment scope from the GitLab group before redeploying.
 
-## Step5: Commit changes to verify the pipeline
+- ### What's gitlab-terraform?
 
+    - `gitlab-terraform` is a thin wrapper around the `terraform` binary as part of the [GitLab Terraform docker image](registry.gitlab.com/gitlab-org/terraform-images/stable:latest) used in `.gitlab-ci.yml`.
```

Comment:
Can you update the link from registry.gitlab.com/gitlab-org/terraform-images/stable:latest to https://gitlab.com/gitlab-org/terraform-images? For context: https://github.com/aws-samples/aws-eks-accelerator-for-terraform/pull/212#discussion_r790006559